services/github/src/handlers.js

```diff
@@ -505,13 +505,28 @@ async function statusHandler(message) {
       const res = await utils.throttleRequest({url, method: 'GET'});
 
       if (res.status >= 400 && res.status !== 404) {
+        let errorMessage = "Failed to get your artifact. ";
+        switch (res.status) {
+          case 403:
+            errorMessage.concat("Make sure your artifact is public. See the documentation on the artifact naming.");
```

Comment:
maybe these error messages should have a `\n` at the beginning of them to visually separate them from the rest of the error message